src/ng/q.js

```diff
@@ -223,22 +222,62 @@ function $QProvider() {
 
 function $$QProvider() {
   this.$get = ['$browser', '$exceptionHandler', function($browser, $exceptionHandler) {
-    return qFactory(function(callback) {
+    return qFactory(noop, function(callback, data) {
       $browser.defer(callback);
     }, $exceptionHandler);
   }];
 }
 
 /**
+ * @ngdoc service
+ * @name $qRaf
```

Comment:
Is this a "public" API? Do application developers need to know about this service or use it in their apps?

Would it be safer to keep it as undocumented semi-private for now?